front-end/legeno-around-here/src/components/AreaSearch.js

```diff
@@ -120,17 +133,24 @@ const AreaSearch = ({ history, selected }) => {
         <Fade in={open}>
           <div className={classes.paper}>
             <h2 id='transition-modal-title'>지역을 검색해주세요!</h2>
-            <TextField
-              id='outlined-search'
-              label='Search field'
-              type='search'
-              variant='outlined'
-              onChange={(event) => getInputArea(event)}
-              inputProps={{ maxLength: 40 }}
-            />
-            <Button>
-              <SearchIcon onClick={() => searchAreaKeyWord()} />
-            </Button>
+            <div>
+              <TextField
+                id='outlined-search'
+                label='Search field'
+                type='search'
+                variant='outlined'
+                onChange={(event) => getInputArea(event)}
+                inputProps={{ maxLength: 40 }}
+              />
+              <Button>
+                <SearchIcon onClick={() => searchAreaKeyWord()} />
+              </Button>
+            </div>
+            <div className={classes.internal}>
+              <Button variant='contained' color='primary' onClick={() => setMainAreaAll()}>
```

Comment:
```suggestion
              <Button variant='contained' color='primary' onClick={setMainAreaAll}> 
```

이렇게 써도 되는 걸로 아는데, 혹시 의도하시고 `=>` 표현 방식을 사용하신건지 궁급하니다!